test/locales/translations.test.js

```diff
@@ -133,6 +133,8 @@ const ignoredTransKeyForYourLoops = [
   "accompanying-documents",
   "intended-use",
   "training",
+  "birthdate",
```

Comment:
Don't forget to remove it from this list if you refactor the translation keys (cf first comment)